### PR TITLE
refine/text classification tester

### PR DESCRIPTION
**Pull request by Superjomn**
- code clean

### CODE REVIEW

File: paddle/fluid/inference/analysis/CMakeLists.txt [hunk 1]
```diff
@@ -105,6 +105,6 @@ if (NOT EXISTS ${TEXT_CLASSIFICATION_INSTALL_DIR} AND WITH_TESTING AND WITH_INFE
   inference_download_and_uncompress(${TEXT_CLASSIFICATION_INSTALL_DIR} ${TEXT_CLASSIFICATION_MODEL_URL} "text-classification-Senta.tar.gz")
 endif()
 
-inference_analysis_test(test_text_classification SRCS test_text_classification.cc
+inference_analysis_test(test_text_classification SRCS analyzer_text_classification_tester.cc
```
Comment (luotao1) [inline paddle/fluid/inference/analysis/CMakeLists.txt:108]: target名字test_text_classification->test_analyzer_text_classification

File: paddle/fluid/inference/analysis/CMakeLists.txt [hunk 2]
```diff
@@ -105,6 +105,6 @@ if (NOT EXISTS ${TEXT_CLASSIFICATION_INSTALL_DIR} AND WITH_TESTING AND WITH_INFE
   inference_download_and_uncompress(${TEXT_CLASSIFICATION_INSTALL_DIR} ${TEXT_CLASSIFICATION_MODEL_URL} "text-classification-Senta.tar.gz")
 endif()
```
Comment (luotao1) [inline paddle/fluid/inference/analysis/CMakeLists.txt:106]:
```
-- Download inference test stuff text-classification-Senta.tar.gz from http://paddle-inference-dist.bj.bcebos.com/text-classification-Senta.tar.gz
tar: Ignoring unknown extended header keyword `SCHILY.dev'
tar: Ignoring unknown extended header keyword `SCHILY.ino'
tar: Ignoring unknown extended header keyword `SCHILY.nlink'
tar: Ignoring unknown extended header keyword `SCHILY.dev'
tar: Ignoring unknown extended header keyword `SCHILY.ino'
tar: Ignoring unknown extended header keyword `SCHILY.nlink'
tar: Ignoring unknown extended header keyword `SCHILY.dev'
tar: Ignoring unknown extended header keyword `SCHILY.ino'
tar: Ignoring unknown extended header keyword `SCHILY.nlink'
tar: Ignoring unknown extended header keyword `SCHILY.dev'
```
可能是因为在mac上打包的原因。https://github.com/yarnpkg/yarn/issues/770#issuecomment-309219076

**Review by luotao1 — APPROVED** [review]

File: paddle/fluid/inference/analysis/CMakeLists.txt (hunk 1 above)
Comment (Superjomn) [reply paddle/fluid/inference/analysis/CMakeLists.txt:108]: 嗯

File: paddle/fluid/inference/analysis/CMakeLists.txt (hunk 2 above)
Comment (sfraczek) [reply paddle/fluid/inference/analysis/CMakeLists.txt:106]: I got the same error

**Pull request merged**